### ISSUE TITLE
workorders.lic - Search for Wood Stain instead of Stain

### ISSUE THREAD

**Issue by dakre18** [rpherbig/dr-scripts#3988]
When the script searches for stain, it finds more than wood stain, which may throw off the script.

> [workorders]>inv search stain
> You rummage about your person, looking for something...
>
>  Your flame-bladed zweihander is in a mottled green leather weapon harness intricately beaded in shades of gold.
 > Your wood stain is in a large hunting pack crafted from wyvern hide.
 > Your jadeleaf anklet.

**Comment by rpherbig** [comment]
Does `inv search wood stain` work?

**Comment by dakre18** [comment]
yes, here's the difference.

>inv search wood stain
>You rummage about your person, looking for something...
>
>  Your wood stain is in a large hunting pack crafted from wyvern hide.

**Comment by dakre18** [comment]
I looked at the stains from HE, and remembered this change. Using those stains may be a problem when searching for "Wood Stain" because they are Charcoal-grey stain, Saffron stain, and Pale-brown stain. I checked with a Charcoal-grey stain in my pack, and it only sees the storebought wood stain (when i use INV SEARCH WOOD STAIN).

So i think i understand why stain was originally used, but i wanted to point this out in case the script needs adjusted again.
